query-translator/src/translate.rs

```diff
@@ -68,6 +73,40 @@ impl ToConstraint for ColumnConstraint {
                 Constraint::equal(qa.to_column(), ColumnOrExpression::Value(tv)),
             EqualsColumn(left, right) =>
                 Constraint::equal(left.to_column(), right.to_column()),
+            EqualsPrimitiveLong(table, value) => {
+                let value_column = QualifiedAlias(table.clone(), DatomsColumn::Value).to_column();
+                let tag_column = QualifiedAlias(table, DatomsColumn::ValueTypeTag).to_column();
+                Constraint::And {
+                    constraints: vec![
+                        Constraint::equal(value_column, ColumnOrExpression::Value(TypedValue::Long(value))),
+                        Constraint::In {
+                            left: tag_column,
+                            list:
+                                if value < 0 {
```

Comment:
This is not wrong, but it's the first time that we've used this information (both for refs and for booleans).  We've been consistent for booleans that `false` is 0, but we haven't used that `true` is 1 AFAIK -- we've always used that `true` is not 0.  No change needed, just noting this.